modules/system/console/OctoberInstall.php

```diff
@@ -291,7 +311,11 @@ protected function setupMigrateDatabase()
 
         try {
             Db::purge();
-            UpdateManager::instance()->resetNotes()->update();
+
+            UpdateManager::instance()
+                ->setNotesOutput($this->output)
```

Comment:
Please explain why the change from `resetNotes()` to `setNotesOutput($this->output)` is required